nav2_bt_navigator/include/nav2_bt_navigator/navigate_to_pose_behavior_tree.hpp

```diff
@@ -20,6 +20,8 @@
 #include "nav2_tasks/behavior_tree_engine.hpp"
 #include "nav2_tasks/follow_path_task.hpp"
 #include "nav2_tasks/global_localization_service_client.hpp"
+#include "nav2_tasks/clear_entirely_costmap_service_client.hpp"
+#include "nav2_msgs/srv/clear_entire_costmap.hpp"
```

Comment:
I recall you wanted to replace this custom srv with the std empty srv. If not in this PR perhaps you could submit an issue and follow-up

Reply:
Yes, will do it in another PR.